src/toast/toast.ts

```diff
@@ -97,17 +115,25 @@ export class NgbToast implements AfterContentInit,
    * Additionally this output is purely informative. The toast won't disappear. It's up to the user to take care of
    * that.
    */
-  @Output('hide') hideOutput = new EventEmitter<void>();
+  @Output('hidden') hiddenOutput = new EventEmitter<void>();
 
-  constructor(@Attribute('aria-live') public ariaLive: string, config: NgbToastConfig) {
+  constructor(
+      @Attribute('aria-live') public ariaLive: string, config: NgbToastConfig, private _zone: NgZone,
+      private _element: ElementRef) {
     if (this.ariaLive == null) {
       this.ariaLive = config.ariaLive;
     }
     this.delay = config.delay;
     this.autohide = config.autohide;
+    this.animation = config.animation;
   }
 
-  ngAfterContentInit() { this._init(); }
+  ngAfterContentInit() {
+    this._zone.onStable.asObservable().pipe(take(1)).subscribe(() => {
```

Comment:
Why do we need zone here ?

Reply:
I guess we need to add `.show` class when `.fade` is already in place, otherwise the browser won't triggger the fade in transition. And we need this to be synchronous.

Reply:
Exactly. No animation is shown without it.